iseq.c

```diff
@@ -2210,6 +2210,12 @@ rb_iseq_disasm_recursive(const rb_iseq_t *iseq, VALUE indent)
 	    if (*argi) rb_str_catf(str, "<%s>", argi);
 	    if ((width -= RSTRING_LEN(str)) > 0) rb_str_catf(str, "%*s", (int)width, "");
 	}
+	if (body->param.flags.has_opt) {
+	    char opti[0x100] = "";
+	    snprintf(opti, sizeof(opti), "Opt=%"PRIdVALUE,
+		    body->param.opt_table[body->local_table_size]);
+	    rb_str_catf(str, "[ 0] <%s>", opti);
```

Comment:
You can use `rb_str_catf` at once, without the intermediate buffer.

```c
rb_str_catf(str, "[ 0] <Opt=%"PRIdVALUE">", body->param.opt_table[body->local_table_size]);
```

Reply:
thx! fixed at 0b8b884e7c12738206e1dc6085ffc775721ff45f